### DISCUSSION TITLE
Dynamic Parallel Routes with url parameters mutation issue 404 Error

### DISCUSSION THREAD

**Discussion by step2341** [vercel/next.js#64764] [Help]
### Summary

Hi All,

I'm having an issue attempting to use Parallel Routes with Dynamic Routes with URL parameters.  I have a use case where i need to render a table in a modal on its own route. In addition, I need to make use of an API's server side sorting and filtering capabilities to filter and sort the table contents. Everything in this project is running on dynamic routes, so i was able to get a modal to render using something like:

src\app\@modal\(.)project\[projectSlug]\detail   (note backslashes are disappearing in the github post on save, but you guys know what i mean.)

problem is though with server side pagination the URL mutates as pagination and sorting params are appended, i.e. ( ?page_1 etc.). When this occurs a 404 results.  

How should we go about appending params to a parallel route?

**Comment by hosain-mohamed** [comment]
I have same issue
